docs/DifferentiableProgramming.md

```diff
@@ -1696,48 +1696,49 @@ public protocol ElementaryFunctions {
     ...
 }
 
-public extension ElementaryFunctions where Self: Differentiable, Self == Self.TangentVector {
+public extension ElementaryFunctions
+where Self: Differentiable & FloatingPoint, Self == Self.TangentVector {
```

Comment:
Note: the `FloatingPoint` requirement here is unfortunate - it's necessary for operations like `*` and `/`.
No general protocol defines these operations yet.